library/WebUSB/WebUSB.h

```diff
@@ -63,6 +69,7 @@ class WebUSB : public PluggableUSBModule, public Stream
 	void begin(unsigned long);
 	void begin(unsigned long, uint8_t);
 	void end(void);
+	void setShortName(const char* name);
```

Comment:
Can you add a comment explaining when this function should be called (during `setup()`) and the restrictions on the parameter (static string, 20 characters or less depending on other pluggable USB modules).

Reply:
Good point, done.